packages/components/navbar/src/utils.styles.ts

```diff
@@ -0,0 +1,7 @@
+type screens = 'medium' | 'large' | 'xlarge';
+type mediaqueries = Record<screens, string>;
+export const mqs: mediaqueries = {
+  medium: '@media (min-width: 480px)',
+  large: '@media (min-width: 768px)',
+  xlarge: '@media (min-width: 1920px)',
+};
```

Comment:
Yes, it makes sense. Do you remember why we didn't do it in the first place?

Reply:
Uhmm not really, but maybe because it was only used inside the navigation at the time, but we may want to use it elsewhere, or maybe the sizes should even be tokens, so it could be used more easily?? WDYT??